### ISSUE TITLE
error running example

### ISSUE THREAD

**Issue by Uniky** [Limitex/ComfyUI-Diffusers#15]
Error occurred when executing DiffusersPipelineLoader:

File "E:\StableDiffusion\ComfyUI_windows_portable\ComfyUI\execution.py", line 154, in recursive_execute
output_data, output_ui = get_output_data(obj, input_data_all)
File "E:\StableDiffusion\ComfyUI_windows_portable\ComfyUI\execution.py", line 84, in get_output_data
return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
File "E:\StableDiffusion\ComfyUI_windows_portable\ComfyUI\execution.py", line 77, in map_node_over_list
results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
File "E:\StableDiffusion\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Diffusers\nodes.py", line 31, in create_pipeline
StableDiffusionPipeline.from_single_file(
File "E:\StableDiffusion\ComfyUI_windows_portable\python_embeded\lib\site-packages\diffusers\pipelines\pipeline_utils.py", line 708, in save_pretrained
save_method(os.path.join(save_directory, pipeline_component_name), **save_kwargs)
File "E:\StableDiffusion\ComfyUI_windows_portable\python_embeded\lib\site-packages\diffusers\models\modeling_utils.py", line 520, in save_pretrained
safetensors.torch.save_file(
File "E:\StableDiffusion\ComfyUI_windows_portable\python_embeded\lib\site-packages\safetensors\torch.py", line 281, in save_file
serialize_file(_flatten(tensors), filename, metadata=metadata)
File "E:\StableDiffusion\ComfyUI_windows_portable\python_embeded\lib\site-packages\safetensors\torch.py", line 475, in _flatten
return {
File "E:\StableDiffusion\ComfyUI_windows_portable\python_embeded\lib\site-packages\safetensors\torch.py", line 479, in
"data": _tobytes(v, k),
File "E:\StableDiffusion\ComfyUI_windows_portable\python_embeded\lib\site-packages\safetensors\torch.py", line 439, in _tobytes
return data.tobytes()

**Comment by Limitex** [comment]
Thank you for the report. Maybe it's an error when loading the model?
I think the possibility is that the browser was not updated when the model was deleted from the directory.

**Comment by Uniky** [comment]
Yes, the error occurs in the Diffusers Pipeline Loader. I have uninstalled and reinstalled this node, still got this error.

**Comment by MattFog** [comment]
where are we supposed to save / how do we set the path for the diffusers model folders?
